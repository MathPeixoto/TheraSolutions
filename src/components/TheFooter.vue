<template>
  <footer class="d-flex align-center justify-space-between">
    <div class="d-flex align-center">
      <v-radio-group class="ml-3 default-size" v-model="radioSelected">
        <v-radio
            color="#FF8700"
            value="selected"
        >
          <template v-slot:label>
            <span id="live">NO AR</span>
          </template>
        </v-radio>
      </v-radio-group>
      <div class="d-flex align-center">
        <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
          <path style="fill: #666666"
                d="M21 3H3c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h5v2h8v-2h5c1.1 0 1.99-.9 1.99-2L23 5c0-1.1-.9-2-2-2zm0 14H3V5h18v12z"/>
        </svg>
        <span class="ml-2 appointment default-size">ENCONTRO - 10:00</span>
        <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
          <path style="fill: #666666"
                d="M21 10.12h-6.78l2.74-2.82c-2.73-2.7-7.15-2.8-9.88-.1-2.73 2.71-2.73 7.08 0 9.79 2.73 2.71 7.15 2.71 9.88 0C18.32 15.65 19 14.08 19 12.1h2c0 1.98-.88 4.55-2.64 6.29-3.51 3.48-9.21 3.48-12.72 0-3.5-3.47-3.53-9.11-.02-12.58 3.51-3.47 9.14-3.47 12.65 0L21 3v7.12zM12.5 8v4.25l3.5 2.08-.72 1.21L11 13V8h1.5z"/>
        </svg>
        <span class="ml-2 update">Última atualização em 10:28</span>
      </div>
    </div>
    <div class="d-flex align-center">
      <div class="d-flex align-center">
        <span
            class="date default-size">Quinta, <strong>30</strong> de <strong>Maio</strong> de <strong>2019</strong></span>
        <span class="ml-2 mr-2 default-size time">// 10:30:42</span>
      </div>
    </div>
  </footer>
</template>

<script>
export default {
  name: "TheFooter",
  data() {
    return {
      radioSelected: "selected",
    }
  }
}
</script>

<style lang="scss" scoped>
footer {
  background-color: #333333;
  width: 100%;
  position: absolute;
  bottom: 0;
  left: 0;
}

.default-size {
  width: 175px;
}

.appointment {
  color: #E9E9E9;
  font-family: "Roboto", sans-serif;
  font-weight: 500;
  font-size: 14px;
}

.update {
  color: #E9E9E9;
  font-family: "Roboto", sans-serif;
  font-weight: 400;
  font-style: italic;
  font-size: 14px;
}

.date {
  color: #E9E9E9;
  font-family: "Roboto", sans-serif;
  font-weight: 300;
  font-style: italic;
  font-size: 14px;
}

.time {
  color: #FF8700;
  font-family: Audiowide, cursive;
  font-weight: 400;
  font-size: 24px;
}

#live {
  color: #FF8700;
  font-family: Audiowide, cursive;
  font-weight: 400;
  font-size: 12px;
}
</style>