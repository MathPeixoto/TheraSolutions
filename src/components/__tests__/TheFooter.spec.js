import { mount, createLocalVue } from "@vue/test-utils";
import Vuetify from "vuetify";
import Vue from "vue";
import TheFooter from "@/components/TheFooter.vue";

Vue.use(Vuetify);
const localVue = createLocalVue();

describe("TheFooter.vue", () => {
  let wrapper;
  let vuetify;

  beforeEach(() => {
    vuetify = new Vuetify();
  });

  afterEach(() => {
    wrapper.destroy();
  });

  it("test if the footer is being well displayed", () => {
    wrapper = mount(TheFooter, {
      localVue,
      vuetify,
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
