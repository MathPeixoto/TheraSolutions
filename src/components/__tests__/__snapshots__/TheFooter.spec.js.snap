// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheFooter.vue test if the footer is being well displayed 1`] = `
<footer
  class="d-flex align-center justify-space-between"
>
  <div
    class="d-flex align-center"
  >
    <div
      class="v-input ml-3 default-size v-input--is-label-active v-input--is-dirty theme--light v-input--selection-controls v-input--radio-group v-input--radio-group--column"
    >
      <div
        class="v-input__control"
      >
        <div
          class="v-input__slot"
          style="height: auto;"
        >
          <div
            aria-labelledby="input-2"
            class="v-input--radio-group__input"
            role="radiogroup"
          >
            <div
              class="v-radio theme--light v-item--active"
            >
              <div
                class="v-input--selection-controls__input"
              >
                <i
                  aria-hidden="true"
                  class="v-icon notranslate mdi mdi-radiobox-marked theme--light"
                  style="color: rgb(255, 135, 0); caret-color: #FF8700;"
                />
                <input
                  aria-checked="true"
                  id="input-3"
                  name="radio-2"
                  role="radio"
                  type="radio"
                  value="selected"
                />
                <div
                  class="v-input--selection-controls__ripple"
                  style="color: rgb(255, 135, 0); caret-color: #FF8700;"
                />
              </div>
              <label
                class="v-label theme--light"
                for="input-3"
                style="left: 0px; position: relative;"
              >
                <span
                  id="live"
                >
                  NO AR
                </span>
              </label>
            </div>
          </div>
        </div>
        <div
          class="v-messages theme--light"
        >
          <transition-group-stub
            class="v-messages__wrapper"
            name="message-transition"
            tag="div"
          />
        </div>
      </div>
    </div>
     
    <div
      class="d-flex align-center"
    >
      <svg
        height="24"
        viewBox="0 0 24 24"
        width="24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M21 3H3c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h5v2h8v-2h5c1.1 0 1.99-.9 1.99-2L23 5c0-1.1-.9-2-2-2zm0 14H3V5h18v12z"
          style="fill: #666666;"
        />
      </svg>
       
      <span
        class="ml-2 appointment default-size"
      >
        ENCONTRO - 10:00
      </span>
       
      <svg
        height="24"
        viewBox="0 0 24 24"
        width="24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M21 10.12h-6.78l2.74-2.82c-2.73-2.7-7.15-2.8-9.88-.1-2.73 2.71-2.73 7.08 0 9.79 2.73 2.71 7.15 2.71 9.88 0C18.32 15.65 19 14.08 19 12.1h2c0 1.98-.88 4.55-2.64 6.29-3.51 3.48-9.21 3.48-12.72 0-3.5-3.47-3.53-9.11-.02-12.58 3.51-3.47 9.14-3.47 12.65 0L21 3v7.12zM12.5 8v4.25l3.5 2.08-.72 1.21L11 13V8h1.5z"
          style="fill: #666666;"
        />
      </svg>
       
      <span
        class="ml-2 update"
      >
        Última atualização em 10:28
      </span>
    </div>
  </div>
   
  <div
    class="d-flex align-center"
  >
    <div
      class="d-flex align-center"
    >
      <span
        class="date default-size"
      >
        Quinta, 
        <strong>
          30
        </strong>
         de 
        <strong>
          Maio
        </strong>
         de
        
        <strong>
          2019
        </strong>
      </span>
       
      <span
        class="ml-2 mr-2 default-size time"
      >
        // 10:30:42
      </span>
    </div>
  </div>
</footer>
`;
